### PR TITLE
Various UI improvements and a bug fix.

### PR DESCRIPTION
**Pull request by ronickg**
Bug fixes:

- fix the amount and fee values not correctly updating in send page when pressing onSubmit

UI improvements:

- Show addresses in this format now  "txch1fz...agzfzg" which is responsive to the width
- Added masked inputs for fee or amount inputs which prevent inputing a smaller decimal then decimals or default of 12.
- Added a warning to confirmation dialog if the users fee is very high

### CODE REVIEW

File: src-tauri/gen/android/app/build.gradle.kts [hunk 1]
```diff
@@ -25,20 +25,20 @@ android {
...
         versionName = tauriProperties.getProperty("tauri.android.versionName", "1.0")
     }
-    signingConfigs {
-        create("release") {
-            val keystorePropertiesFile = rootProject.file("keystore.properties")
-            val keystoreProperties = Properties()
-            if (keystorePropertiesFile.exists()) {
-                keystoreProperties.load(FileInputStream(keystorePropertiesFile))
-            }
+    // signingConfigs {
```
Comment (Rigidity) [inline src-tauri/gen/android/app/build.gradle.kts:28]: Why is this commented out?

File: src-tauri/gen/android/app/src/main/res/values/themes.xml [hunk 2]
```diff
@@ -1,6 +1,10 @@
 <resources xmlns:tools="http://schemas.android.com/tools">
     <!-- Base application theme. -->
     <style name="Theme.sage_tauri" parent="Theme.MaterialComponents.DayNight.NoActionBar">
+      <!-- <item name="android:statusBarColor">@android:color/transparent</item> -->
```
Comment (Rigidity) [inline src-tauri/gen/android/app/src/main/res/values/themes.xml:4]: What is this for?

File: src-tauri/gen/apple/sage-tauri.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -355,7 +355,7 @@
 				CODE_SIGN_ENTITLEMENTS = "sage-tauri_iOS/sage-tauri_iOS.entitlements";
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = NQJQRYZZG3;
+				DEVELOPMENT_TEAM = 5PSQ3NC8JP;
```
Comment (Rigidity) [inline src-tauri/gen/apple/sage-tauri.xcodeproj/project.pbxproj:358]: Does this need to be changed?

File: src/components/ConfirmationDialog.tsx [hunk 4]
```diff
@@ -127,6 +127,16 @@ export default function ConfirmationDialog({
                   </TabsTrigger>
                 </TabsList>
                 <TabsContent value='simple'>
+                  {/* {isHighFee && !fee.isZero() && ( */}
```
Comment (Rigidity) [inline src/components/ConfirmationDialog.tsx:130]: What is this comment for?

File: src/components/Layout.tsx [hunk 5]
```diff
@@ -43,8 +45,9 @@ export default function Layout(props: PropsWithChildren<object>) {
               to='/wallet'
               className='flex items-center gap-2 font-semibold'
             >
-              <img src={icon} className='h-6 w-6' />
-              {wallet?.name}
+              <img src={icon} className='h-8 w-8' />
+              {/* {wallet?.name} */}
```
Comment (Rigidity) [inline src/components/Layout.tsx:49]: Do we need this still

**Comment by ronickg** [comment]
Went over the comments, removed and reverted not needed changes. Fixed the fee is to high warning to use mojo's now as well after the merge. Also fixed the copybox to have a value and dispValue, otherwise the copy will use the short format address.

**Review by Rigidity — APPROVED** [review]

**Pull request merged**